common/src/main/java/com/wynntils/core/features/Feature.java

```diff
@@ -210,6 +210,10 @@ public boolean canEnable() {
         return true;
     }
 
+    public boolean canUserEnable() {
+        return getConfigOptionFromString("userEnabled").isPresent();
```

Comment:
Seem a bit backwards to determine this from config strings...

Reply:
This could be an instaceof check too, yes.

Reply:
I'd prefer that.